assets/src/amp-validation-tooltips.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+// WIP Pointer function
+function sourcesPointer() {
+	jQuery( '.tooltip' ).on( 'hover', function() {
```

Comment:
I don't think that `hover` works. The tooltip really needs to not get in the way. It's too easy to trigger it. The problem could be solved if we switch to showing the tooltip on _click_ rather than on hover. This would also make it keyboard accessible.